webapp/channels/src/components/channel_members_rhs/header.tsx

```diff
@@ -17,8 +17,11 @@ interface Props {
     goBack: () => void;
 }
 
-const HeaderTitle = styled.span`
+const HeaderTitle = styled.h2`
```

Comment:
same here, but feel free to totally ignore it this time, just bringing awareness about it.